content/influxdb/v2/process-data/common-tasks/downsample/downsample-data-quix.md

```diff
@@ -0,0 +1,321 @@
+---
```

Comment:
Move this page to `content/influxdb/v2/tools/downsample-data-quix.md`.

Reply:
Hi @sanderson I have moved it but think it's better in common tasks. Thats where the other downsampling examples are. You are right in your other comments though. I have have confused the issue by talking about v2 -> downsample -> v3. So I will ensure all the pages use v2 OR v3 and not both.